common/socketoptions.hpp

```diff
@@ -142,36 +173,39 @@ inline bool SocketOption::apply(int socket, std::string value) const
         SRT_HANDLE_TYPE(INT);
         SRT_HANDLE_TYPE(INT64);
         SRT_HANDLE_TYPE(BOOL);
+        SRT_HANDLE_TYPE(ENUM);
 
 #undef SRT_HANDLE_TYPE
     }
     return false;
 }
 
+extern std::map<std::string, int> enummap_transtype;
```

Comment:
Ups, sorry, my mistake. This should have been moved to the next commit, which will add a new option. Fortunately this isn't used anywhere for the time being.